src/helpers/generateTVArticle/GenerateListTvOptions.jsx

```diff
@@ -1,28 +1,25 @@
-import checkIcon from "../assets/check.png";
-import notIcon from "../assets/minus.png";
+// part 2
+import checkIcon from "../../assets/check.png";
+import notIcon from "../../assets/minus.png";
 
 function GenerateListTvOptions(tv) {
-    let listTvOptions = [];
-    let counter = 0;
-    for (const option of tv.options) {
+    return tv.options.map((option) => {
         if (option.applicable === true) {
-            listTvOptions.push(<li key={counter}>
+            return <li key={option}>
```

Comment:
option is hier nog een object met de twee keys 'name' en 'applicable'. Een object is geen geldige key. Zorg ervoor dat je altijd een unieke key meegeeft, in dit geval was dat helaas niet mogelijk dus dan mag je gebruik maken van de index. Zo dus: 

`    return tv.options.map((option, index) => {
        if (option.applicable === true) {
            listTvOptions.push(<li key={counter}>
            return <li key={index}>`